tensilelite/Tensile/Components/LocalRead.py

```diff
@@ -250,7 +254,7 @@ def __call__(self, writer, kernel, bufferIdx, iui, epsi, tP):
                 comment = "LDS Transpose"
                 valuiIdx = int(valufIdx)
                 localReadCode = imod.add(Module("LocalRead%s Valu%u"%(tc,valuiIdx)))
-                localReadCode.add(LocalReadX(dst=destVgpr, src=vgpr("LocalReadAddr%s"%tc), ds=ds, comment=comment))
+                localReadCode.add(LocalReadX(dst=destVgpr, src=srcAddr, ds=ds, comment=comment))
                 destVgpr = vgpr("Valu%s_X%u_I%u+%u+2"%(tc,bufferIdx,iui,4*tIdx), 2)
                 offset_val = paramList[0]+UnrollStride*inputPerThread;
```

Comment:
We should apply the same logic as the above for the second LocalReadX.
- keep the original offset value (before -= num * maxLDSConstOffset) for the second one
- do the same calculation (num) here after offset_val = paramList[0]+UnrollStride*inputPerThread
- maybe better to make a small function for srcAddr and offset_val conversion to avoid duplicating the same code